app/page.tsx

```diff
@@ -41,9 +46,59 @@ const filterOptions: FilterOption[] = [
   },
 ];
 
+const nowTimestamp = Math.floor(Date.now() / 1000);
+
+type StateFilterOption = {
+  name: string;
+  key: string;
+  query: string;
+  state: FixedProductMarketMaker_Filter;
+};
+
+const stateFilterOptions: StateFilterOption[] = [
```

Comment:
I'd shorten a bit the naming and change a few things:

* I'd drop "Options" from the variables, using a plural implies it has multiple entries or options, and if we want to specify it's an array structure, we could use "List" instead of "Options".
e.g.: `StateFilters`, `StateFilterList`, `selectedState`
* I'd change "State" to "Status", mostly because I'm used to seeing (in codebases) "state" tied to either DOM or storage state.
e.g.: `StatusFilters`, `StatusFilterList`, `selectedStatus`

^These suggestions could lead to an easier-to-read code being less verbose, but it's just a personal opinion, I have no strong opinions about this specific.

Reply:
Changed the naming. Let me know what you think